requirements/requirements.in

```
@@ -7,6 +7,7 @@ data_tasks
 pyramid-sanity
 
 # 3rd-party packages
+python-jose[cryptography]
```

I wonder if we want to add `cryptography` as a separate top-level dependencies so that Dependabot keeps it up to date for us? Just because I think we might want to get updates for that

```suggestion
python-jose
cryptography
```

---

I'll defer this to the task I added to the board.